corehq/util/context_processors.py

```diff
@@ -257,3 +257,29 @@ def get_demo(request):
             'is_demo_visible': True,
         })
     return context
+
+
+def hubspot_js(request):
```

Comment:
What would you think of making this a helper function, and then using it in `js_api_keys` to delete the API key? It looks like we do something similar for google, and it'd be nice not to have yet another flag being passed down to the js.